src/AElf.OS.Core/Network/Infrastructure/IPeer.cs

```diff
@@ -3,17 +3,17 @@
 using System.Net;
 using System.Threading.Tasks;
 using AElf.OS.Network.Application;
-using AElf.OS.Network.Grpc;
 using AElf.OS.Network.Metrics;
+using AElf.OS.Network.Protocol.Types;
 using AElf.Types;
 
 namespace AElf.OS.Network.Infrastructure
 {
     public interface IPeer
     {
-        bool IsBest { get; set; }
         bool IsReady { get; }
-        
+
+        Hash LastKnownLibHash { get; }
         long LastKnownLibHeight { get; }
         IPEndPoint RemoteEndpoint { get; }
         string IpAddress { get; }
```

Comment:
IpAddress not assign and only used by logging

Reply:
Sam already removed it in his PR.